techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -115,6 +125,8 @@ bundle agent doInventory_always
       "any" usebundle => moveInventoryToFinalDestination;
       "any" usebundle => signInventories;
       "any" usebundle => compressInventories;
+
+    uuid_valid.(force_inventory|((inventory_time|initial_promises).!inventory_sent)).inventory_valid|resend_inventory::
```

Comment:
shouldn't that be
uuid_valid.(force_inventory|resend_inventory|((inventory_time|initial_promises).!inventory_sent)).inventory_valid
otherwise it will try to send again inventory even if inventory is invalid

Reply:
ping @peckpeck